recipes/bidsschematools/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "bidsschematools" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a091b6a16ec01be00db76209b9a4d19005589382563a6caf503aa1140317a06f
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - bst = bidsschematools.__main__:cli
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+  run:
+    - python >=3.7
+    - click
+    - pyyaml
+    - importlib-resources
+    - tabulate
+    - pandas
+
+test:
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - pyproject.toml
+  imports:
+    - bidsschematools
+  commands:
+    - pip check
+    - bst --help
+    - python -m pytest -vv --pyargs bidsschematools
+
+about:
+  home: https://github.com/bids-standard/bids-specification
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Python tools for working with the BIDS schema
+  dev_url: https://github.com/bids-standard/bids-specification
+
+extra:
+  recipe-maintainers:
+    - jwodder
+    - yarikoptic
+    - TheChymera
```

If you don't mind...

```suggestion
    - TheChymera
    - effigies
```

---

not at all! @jwodder please add @effigies to this honorable list ;)